lib/cisco_node_utils/cmd_ref/itd_device_group.yaml

```diff
@@ -0,0 +1,83 @@
+# itd_device_group
+---
+_exclude: [N3k, N5k, N6k, N8k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd device-group <name>/'
+      - '(?)/^node <ntype> <nname>/'
+  set_context: ["itd device-group <name>", "(?)node <ntype> <nname>"]
+
+all_itd_device_group_nodes:
+  multiple:
+  get_context: ['/^itd device-group <name>/']
+  get_value: '/^node (.*)/'
```

Comment:
I think this could be cleaner if you match on both node afi and node address.

get_value: '/^node (\S+) (\S+)/'

This will allow you to streamline this code a bit more.

https://github.com/cisco/cisco-network-node-utils/pull/274/files#diff-c621588334e7906861d7071affb1c0abR56